compiler/codegen/codegen.cpp

```diff
@@ -2310,7 +2310,7 @@ void codegen() {
     if(fIncrementalCompilation)
       USR_FATAL("Incremental compilation is not yet supported with LLVM");
 
-    if(printCppLineno || debugCCode)
```

Comment:
Do you intend to remove this in this PR?

I am not sure whether it's useful anyway with llvm backend

Reply:
Yes, because when I was comparing the LLVM IR in the process of debugging I saw quite a lot of debug information that clang wasn't emitting (i.e. I think we should only emit that information with `-g`; in contrast `printCppLineno` is on by default).